astropy/coordinates/tests/test_name_resolve.py

```diff
@@ -116,7 +116,7 @@ def test_names():
         NameResolveError,
         match=re.escape(
             "Unable to find coordinates for name 'm87h34hhh' "
-            "using https://cds.unistra.fr/cgi-bin/nph-sesame/SNV?m87h34hhh"
+            "using http://vizier.cfa.harvard.edu/viz-bin/nph-sesame/SNV?m87h34hhh"
```

Comment:
Maybe we need to be agnostic to the exact web site - this might depend on where the request came from (since vizier really is based at CDS, the harvard one is a mirror).

Reply:
yes, that's probably best. Let me simplify this regexp